src/docs/source/includes/_deployment.md

```diff
@@ -520,4 +522,13 @@ directly to the `edge` service resource:
     ```
 
     > Locating `EXTERNAL-IP` for Ingress Controller. **NOTE** this will be
-    [localhost](https://localhost) on local clusters.
\ No newline at end of file
+    [localhost](https://localhost) on local clusters.
```

Comment:
There's a clear tag saying "Istio" for that option?

Reply:
what do you mean?

Reply:
Just asking if there’s a separation of “with isitio” and “without sítio”. I saw that there’s a note “if you are using istio”. 

I didn’t rendered the docs yet, do not know how is showing.

Reply:
Istio is under a separate heading, however, since I am referring back to the deployment I figured to add the "If using Istio" just in case.